docs/develop/processing.md

```diff
@@ -32,17 +73,18 @@ def preprocess(self, model_input: Any) -> Any:
     return np.array(preprocessed_image)
 ```
 
-## Post-processing
+### Post-processing
 
 Similarly, the output of a ML model can be messy or cryptic. Demystify the model results for your end user or format a web-friendly response in the post-processing function.
 
+**Example: ResNet post-processing**
+
 Here is a post-processing function from the same [TensorFlow example](../create/tensorflow.md) that returns clean labels.
 
 ```python
 from scipy.special import softmax
 
 def postprocess(self, model_output: Any, k=5) -> Any:
```

Comment:
this has the wrong model signature. You shouldn't have `k` like that. it works but isn't a good example that people should follow